docs/en/ingest-management/fleet/add-fleet-server-kubernetes.asciidoc

```diff
@@ -0,0 +1,392 @@
+
+. Open the created policy, and from the *Integrations* tab select *Add integration*:
++
+** Search and select the Fleet Server integration
+** Select *Add Fleet Server* to add the integration to the policy.
++
+At this point you can configure the integration settings per <<fleet-server-scalability>>.
+** When done, select on *Save and continue*. Do not add any {agent} at this stage.
+
+. Open the configured policy, which now includes the Fleet Server integration, and select *Actions* → *Add Fleet Server*. In the next dialog:
++
+* Confirm that the *policy for Fleet Server* is properly selected.
+* *Choose a deployment mode for security*:
+** If you select *Quick start*, the {fleet-server} generates a `self-signed` TLS certificate, and subsequent agents should be enrolled using the `--insecure` flag.
+** If you select *Production*, you provide a certificate and key to the {fleet-server} during the deployment, and subsequent agents will need to trust the certificate's CA.
+* Add your *Fleet Server Host* information:
+** This will be the URL used by the clients ({agent}s) to connect to the {fleet-server}. In *Production* mode, the {fleet-server} certificate must include the hostname part of the URL as an `x509 SAN`.
+** On Kubernetes environments this could be the name of the `Kubernetes service` or reverse proxy that exposes the {fleet-server} Pods.
+** This setting is not really used by the {fleet-server} itself, so don't worry much about it at this stage because you will be able to change it later on under *Fleet → Settings*.
+* Select **generate service token** to create a token for the {fleet-server}.
+* From *Install Fleet Server to a centralized host → Linux*, take note of the values of the following settings that will be needed for the {fleet-server} installation:
+** Service token(under `--fleet-server-service-token` parameter).
+** Fleet policy ID (under `--fleet-server-policy` parameter).
+** {es} URL (under `--fleet-server-es` parameter).
+
+. Keep {kib} browser window opened and continue with the <<add-fleet-server-kubernetes-install>>.
++
+When {fleet-server} installation succeeds, the *Confirm Connection* dialog will show *Connected*.
+
+[discrete]
+[[add-fleet-server-kubernetes-install]]
+== Fleet Server installation
+
+To deploy {fleet-server} on Kubernetes and register it into Fleet you need the following details:
+
+* *Policy ID* of the Fleet policy configured with the {fleet-server} integration.
+* *Service token*, that you can generate following the <<add-fleet-server-kubernetes-preparations>> or manually using the {ref}/service-tokens-command.html[{es}-service-tokens command].
+* *{es} endpoint URL*, configured also in the {es} output associated to the policy.
+* *{es} CA certificate file*, configured also in the {es} output associated to the policy.
```

Comment:
I'd probably remove `also` in these lines. Without it, it reads to me as "You'll need this setting, and here's where you can find it".

Reply:
> Without it, it reads to me as "You'll need this setting, and here's where you can find it".

mmmm no, the objective is to remember the users that they need the ES endpoint URL configured in BOTH places (same for the CA certificate). We are not telling the users from where to obtain the info (except for the service-token).

And actually, the ES outuput in Fleet UI will appear by default with an "invalid" URL like `http://localhost:9200`, and the CA information will be empty.

What I try to remind the users (maybe too many times) is that both the CA and the URL need to be added in two different places.

But it's true that this isn't intuitive in the current doc text.... what about this?

> *{es} endpoint URL*, configured in both the {es} output associated to the policy and in the Fleet Server as an environment variable.
> *{es} CA certificate file*, configured in both the {es} output associated to the policy and in the Fleet Server.

Let me know your thoughts @kilfoyle, and thanks! ;)

Reply:
Thanks for explaining Edu! I really like the the proposed text.

This seems 100% clear:

> *{es} endpoint URL*, configured in both the {es} output associated to the policy and in the Fleet Server as an environment variable.
> *{es} CA certificate file*, configured in both the {es} output associated to the policy and in the Fleet Server.